.github/CONTRIBUTORS.MD

```diff
@@ -3,4 +3,5 @@
 Specials thanks to the members of Stevens Blueprint for collaborating on this project
 
 <!--Please add your name (First Name Last Name)-->
-- [Miguel Merlin](https://github.com/miguel-merlin)
\ No newline at end of file
+- [Miguel Merlin](https://github.com/miguel-merlin)
+- [Phillip Anerine](https://github.com/phill52)
```

Comment:
Please, for next time, try to keep the changes in pr atomic. Meaning try to avoid unrelated changes out of the pr. https://mtlynch.io/code-review-love/